apps/fulfillment/sw/main.ts

```diff
@@ -1,5 +1,7 @@
 /// <reference lib="WebWorker" />
 
+import './window-stub';
```

Comment:
Can you add a line of explanation, why are we stubbing window in service worker?